auth/range_perm_cache_test.go

```diff
@@ -123,7 +127,7 @@ func TestGetMergedPerms(t *testing.T) {
 	for i, tt := range tests {
 		result := mergeRangePerms(tt.params)
 		if !isPermsEqual(result, tt.want) {
-			t.Errorf("#%d: result=%q, want=%q", i, result, tt.want)
+			t.Fatalf("#%d: result=%q, want=%q", i, result, tt.want)
```

Comment:
we should change back to errorf. if there is one case fails, we should continue to execute the next case. then we can detect multiple failures in one round of test.